geaflow/geaflow-dsl/geaflow-dsl-connector/geaflow-dsl-connector-jdbc/src/main/java/com/antgroup/geaflow/dsl/connector/jdbc/JDBCTableSink.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 AntGroup CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.antgroup.geaflow.dsl.connector.jdbc;
+
+import com.antgroup.geaflow.api.context.RuntimeContext;
+import com.antgroup.geaflow.common.config.Configuration;
+import com.antgroup.geaflow.dsl.common.data.Row;
+import com.antgroup.geaflow.dsl.common.exception.GeaFlowDSLException;
+import com.antgroup.geaflow.dsl.common.types.StructType;
+import com.antgroup.geaflow.dsl.connector.api.TableSink;
+import com.antgroup.geaflow.dsl.connector.jdbc.util.JDBCUtils;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JDBCTableSink implements TableSink {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JDBCTableSink.class);
+
+    private Configuration tableConf;
+    private StructType schema;
+    private String driver;
+    private String url;
+    private String username;
+    private String password;
+    private String tableName;
+    private Connection connection;
+    private Statement statement;
+
+    @Override
+    public void init(Configuration tableConf, StructType tableSchema) {
+        LOGGER.info("prepare with config: {}, \n schema: {}", tableConf, tableSchema);
+        this.tableConf = tableConf;
+        this.schema = tableSchema;
+
+        this.driver = tableConf.getString(JDBCConfigKeys.GEAFLOW_DSL_JDBC_DRIVER);
+        this.url = tableConf.getString(JDBCConfigKeys.GEAFLOW_DSL_JDBC_URL);
+        this.username = tableConf.getString(JDBCConfigKeys.GEAFLOW_DSL_JDBC_USERNAME);
+        this.password = tableConf.getString(JDBCConfigKeys.GEAFLOW_DSL_JDBC_PASSWORD);
+        this.tableName = tableConf.getString(JDBCConfigKeys.GEAFLOW_DSL_JDBC_TABLE_NAME);
+    }
+
+    @Override
+    public void open(RuntimeContext context) {
+        try {
+            Class.forName(this.driver);
+            this.connection = DriverManager.getConnection(url, username, password);
+            this.statement = connection.createStatement();
+            JDBCUtils.createTable(statement, this.tableName, this.schema.getFields());
```

Comment:
As a computing engine, GeaFlow should avoid helping users decide on creating tables whenever possible. Instead, it can build a temporary table or just check if the table exists.